### PR TITLE
Fix include order for cpplint

### PR DESCRIPTION
**Pull request by jacobperron**
Relates to https://github.com/ament/ament_lint/pull/324

### CODE REVIEW

File: include/performance_test_fixture/performance_test_fixture.hpp [hunk 1]
```diff
@@ -16,10 +16,11 @@
 #define PERFORMANCE_TEST_FIXTURE__PERFORMANCE_TEST_FIXTURE_HPP_
 
 #include <benchmark/benchmark.h>
-#include <osrf_testing_tools_cpp/memory_tools/memory_tools.hpp>
 
 #include <atomic>
 
+#include <osrf_testing_tools_cpp/memory_tools/memory_tools.hpp>
```
Comment (clalancette) [inline include/performance_test_fixture/performance_test_fixture.hpp]: Should we change to double-quotes here?

Comment (jacobperron) [reply include/performance_test_fixture/performance_test_fixture.hpp]: I don't know. cpplint is happy either way.

Comment (jacobperron) [reply include/performance_test_fixture/performance_test_fixture.hpp]: Well, maybe if we use double-quotes both old and new versions of cpplint will be happy. I'll give it a try.

**Review by clalancette — COMMENTED** [review]
I'm OK with not changing the quotes.

However, I do think we are going to have to do something about the branches here.  In particular it looks like the `main` branch is used for all of foxy, galactic, and rolling, and this PR should only apply to rolling.  So I think we'll have to branch off.

**Comment by jacobperron** [comment]
Looks like using double-quotes (1d0820f) satisfies both old and new versions of cpplint , so I don't think we need to branch.

**Review by clalancette — APPROVED** [review]

**Pull request merged**